doc/specs/GitHub-discussions-spec.md

```diff
@@ -0,0 +1,87 @@
+# **GitHub discussions spec**
+
+<img align="right" width="200" src="/doc/images/Logo/Vintage/Logo.jpg" />
+
+- **What is it:** Creating better interaction with the community.
+- **Authors:** Aaron Junker ([@aaron-junker](https://github.com/aaron-junker))
+- **Spec Status:** Waiting for approve
+
+## What is GitHub Discussions
+
+GitHub Discussions is a feature that currently is in closed beta status. It will be released until the end of the year. 
+
+It's a new tab in the GitHub repo page where people can start new threads like in a forum.
+
+The threads must be added to a category. The answer feature can be enabled or disabled for categorys. If enabled a person with triage permission or the creator of the thread can mark a reply as answer.
+
+You can sort threads in the overview for categories and for (un-)answered.
+
+One thread can be pinned on the main page of the discussion section. (One with rules or something like that).
```

Comment:
This is wrong! See https://github.com/microsoft/winget-cli/discussions

Reply:
Oh thank you. I didn't know that